blocks/library/cover-image/editor.scss

```diff
@@ -22,10 +22,19 @@
 	}
 
 	&.has-left-content .block-rich-text__inline-toolbar {
-		justify-content: flex-start;
+		display: inline-block;
 	}
 
 	&.has-right-content .block-rich-text__inline-toolbar{
-		justify-content: flex-end;
+		display: inline-block;
 	}
+
+	.editor-block-list__layout {
+		width: 100%;
+		.editor-block-list__block:not(.is-multi-selected) .wp-block-paragraph {
+			background: none;
```

Comment:
Hi @youknowriad,
The reason why we set a white background is related to the contrast verification. If the background was never set getComputedStyles returns back background (if I remember correctly). So adding a background fixes the problem. If themes change the background it also works well because as long as a background is set we are able to retrieve it. The problem happens when no background is set. As we get back background color when background is not set we cannot differentiate and assume is white background because in fact it may be set to black. I will try to research this in more detail maybe there is a way to detect when the background is not set and in that case assume white.
Replicate from comment https://github.com/WordPress/gutenberg/pull/5273#discussion_r176880358.

A refactoring will happen to contrast checker in order to not rely on this rule. So it will be removed for now.